.github/ci-test-connections.yaml

```diff
@@ -52,3 +52,5 @@ connections:
     conn_type: aws
     description: null
     extra: null
+    login: $AWS_ACCESS_KEY_ID
+    password: $AWS_SECRET_ACCESS_KEY
```

Comment:
What changed for us to need to explicitly give these values now?

Reply:
@tatiana  We need to pass the S3 credentials for bigquery to access the remote file.

Reply:
We can use a similar strategy to this:
https://github.com/astronomer/astro-sdk/pull/523/files#diff-b8140a95c509699b028734bb59473e42133480d0d99ae375e7ada290d8fb03bfR204-R205

Without the need to explicitly define them in the connection.

Reply:
@tatiana I'll rebase when PR - [https://github.com/astronomer/astro-sdk/pull/523/](https://github.com/astronomer/astro-sdk/pull/523/files#diff-b8140a95c509699b028734bb59473e42133480d0d99ae375e7ada290d8fb03bfR204-R205) is merged to main.

Reply:
Will make the change.